src/features/user/CompleteSignup/index.tsx

```diff
@@ -321,7 +332,10 @@ export default function CompleteSignup() {
               label={t('donate:country')}
               name="country"
               onChange={(country) => setCountry(country)}
-              defaultValue={defaultCountry}
+              defaultValue={getStoredConfig("loc").postalCode === "T1" || 
+                            getStoredConfig("loc").postalCode === "XX" || 
+                            getStoredConfig("loc").postalCode === "" ?
+                            "" : getStoredConfig("loc").postalCode}
```

Comment:
The registration currently fails for me saying no country is specified. Maybe this change for the country field is just a bug?

Reply:
I now get an error trying to signup (on staging):

<img width="1686" alt="Bildschirmfoto 2021-04-29 um 09 42 18" src="https://user-images.githubusercontent.com/1532418/116517554-6cae7000-a8cf-11eb-98fc-6b4967897eca.png">